### PR TITLE
Adds a new traitor bundle - Synda Major Leage Player

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request
Semi-tested!
New base ball bat that reflects all laser based weaponds and deals spear harm levels
New suit/hat that have some armor and theme fitting to a major player
New stimpack hypo/medpens

## Why It's Good For The Game

Fun baseball gimmic to spice things up, has weight 1, and is a basically alt of a dark lord that dosnt have TK + magic, should be balanced around its main gimmic being only laser proof and with 0 block without other items

## Changelog
:cl:
add: new traitor bundle
/:cl:

### CODE REVIEW

**Comment by YakumoChen** [comment]
it needs baseballs

**Comment by Trilbyspaceclone** [comment]
We do

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 1]
```diff
@@ -62,6 +62,16 @@
 	desc = "A pair of orange rubber boots, designed to prevent slipping on wet surfaces while also drying them."
 	icon_state = "galoshes_dry"
 
+/obj/item/clothing/shoes/sneakers/noslip
+	desc = "A pair of black shoes, they have the souls of galoshes making them unable to be slipped on a wet surface."
```
Comment (cebutris) [inline code/modules/clothing/shoes/miscellaneous.dm]:
```suggestion
	desc = "A pair of black shoes, they have the soles of galoshes making them unable to be slipped on a wet surface."
```

File: code/game/objects/items/weaponry.dm [hunk 2]
```diff
@@ -578,6 +578,12 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 		playsound(turf, 'sound/weapons/effects/batreflect2.ogg', 50, 1)
 	return 1
 
+/obj/item/melee/baseball_bat/ablative/synda
```
Comment (cebutris) [inline code/game/objects/items/weaponry.dm]:
```suggestion
/obj/item/melee/baseball_bat/ablative/syndi
```

File: code/modules/clothing/under/syndicate.dm [hunk 3]
```diff
@@ -67,3 +67,14 @@
 	icon_state = "syndicate_combat"
 	item_color = "syndicate_combat"
 	can_adjust = FALSE
+
+/obj/item/clothing/under/syndicate/baseball
+	name = "major leage, number unknown"
```
Comment (cebutris) [inline code/modules/clothing/under/syndicate.dm]:
```suggestion
	name = "major league, number unknown"
```

File: code/modules/clothing/under/syndicate.dm [hunk 4]
```diff
@@ -67,3 +67,14 @@
 	icon_state = "syndicate_combat"
 	item_color = "syndicate_combat"
 	can_adjust = FALSE
+
+/obj/item/clothing/under/syndicate/baseball
+	name = "major leage, number unknown"
+	desc = "A major leage outfit with the number faded number on the back. Seems rather robust for just a game"
```
Comment (cebutris) [inline code/modules/clothing/under/syndicate.dm]:
```suggestion
	desc = "A major league outfit with the number faded number on the back. Seems rather robust for just a game"
```

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 5]
```diff
@@ -127,6 +127,21 @@
 	else
 		to_chat(usr, "<span class='notice'>It is spent.</span>")
 
+/obj/item/reagent_containers/hypospray/medipen/stimulants
+	name = "illegal stimpack medipen"
+	desc = "A highly illegal medipen do to its load and small injections, allow for five uses before being drained"
```
Comment (cebutris) [inline code/modules/reagents/reagent_containers/hypospray.dm]:
```suggestion
	desc = "A highly illegal medipen due to its load and small injections, allow for five uses before being drained"
```

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 6]
```diff
@@ -127,6 +127,21 @@
...
+/obj/item/reagent_containers/hypospray/medipen/stimulants
+	name = "illegal stimpack medipen"
+	desc = "A highly illegal medipen do to its load and small injections, allow for five uses before being drained"
+	volume = 50
+	amount_per_transfer_from_this = 10
+	list_reagents = list("stimulants" = 50)
+
+/obj/item/reagent_containers/hypospray/medipen/stimulants/baseball
+	name = "the reason the syndicate major league team wins."
+	desc = "They say drugs never win, but look were you are now, then were they are."
```
Comment (cebutris) [inline code/modules/reagents/reagent_containers/hypospray.dm]:
```suggestion
	desc = "They say drugs never win, but look where you are now, then where they are."
```

**Review by cebutris — CHANGES_REQUESTED** [review]
Spelling errors

**Review by silicons — APPROVED** [review]
i'm nerfing/reverting this so fast if murderbones become common because of this.

**Pull request merged**